go1.x/run/aws-lambda-mock.go

```diff
@@ -24,16 +25,21 @@ import (
 func main() {
 	rand.Seed(time.Now().UTC().UnixNano())
 
+	debugMode := flag.Bool("debug", false, "enables delve debugging")
+	delvePath := flag.String("delvePath", "/var/debug/dlv", "path to delve")
```

Comment:
`/var/debug` seems too generic. I don't want the container to break when official Lambda runtime suddenly makes use of this path in the future.

Instead, can you make it something explicit (ugly?) like `/tmp/lambci_debug_files`